kura/org.eclipse.kura.container.orchestration.provider/src/main/java/org/eclipse/kura/container/orchestration/provider/impl/enforcement/AllowlistEnforcementMonitor.java

```diff
@@ -55,11 +55,17 @@ private void enforceAllowlistFor(String containerId) {
 
         Set<String> digestIntersection = this.enforcementAllowlistContent.stream().distinct()
                 .filter(digestsList::contains).collect(Collectors.toSet());
+      
+        if (digestIntersection.isEmpty()) {
+            digestIntersection = this.orchestrationServiceImpl.getContainerInstancesAllowlist().stream().distinct()
+                    .filter(digestsList::contains).collect(Collectors.toSet());
+        }
+
 
         if (!digestIntersection.isEmpty()) {
-            logger.info(ENFORCEMENT_SUCCESS, digestIntersection, containerId);
+            logger.info(ENFORCEMENT_CHECK_SUCCESS, digestIntersection, containerId);
         } else {
-            logger.error(ENFORCEMENT_FAILURE, containerId);
+            logger.error(ENFORCEMENT_CHECK_FAILURE, containerId);
```

Comment:
This should be a `info` level

```suggestion
            logger.info(ENFORCEMENT_CHECK_FAILURE, containerId);
```